myia/basics.py

```diff
@@ -143,3 +142,11 @@ def return_(x):
 def resolve(ns, key):
     """Resolve the name in the given namespace."""
     return ns[key]
+
+
+def concat(*args):
+    """Contatenate the arguments in a tuple.
+
+    This is the same functionally as make_tuple, but used for macros.
+    """
+    return args
```

Comment:
`concat` should concatenate the arguments, i.e. `reduce(operator.add, args, ())`. I'm not sure why you would concatenate the arguments in `apply` rather than have `concat` do it.